### ISSUE TITLE
Refined Storage Index out of Bonds

### ISSUE THREAD

**Issue by hamurci** [AllTheMods/ATM-8#620]
When I try to request(using "+" sign below) required items to craft crushing wheel of create mod, Game disconnects instantly and puts me to main menu and shows that error message in screenshot
![2023-01-08_16 48 06](https://user-images.githubusercontent.com/34452253/211199997-66648d6a-bbd2-451b-a75f-d6aece0185de.png)
![image](https://user-images.githubusercontent.com/34452253/211199962-697bd44e-3f04-4302-98fb-fc12e1065d5c.png)

**Comment by SerenityFire** [comment]
yeah had the same here, that's maybe because you cannot craft 21 things on a 3x3 table.

**Comment by hamurci** [comment]
Refined storage has more than 9 slots when autocrafting. Something must be wrong with packet size limit I guess
